PSIPro.ino

```diff
@@ -11,10 +11,16 @@
  *    
  *  BEFORE BUILDING OR UPLOADING THIS SKETCH, be sure that the config.h and matrices.h files are in the skectch folder. 
  *
- *  Version 1.1
+ *  Version 1.1.01
```

Comment:
Don't use a patch version of 01, I would like to see this be 1.1.1 if we're doing patches, but in general I would like to stick with only Major.Minor release numbering.  As such lets up-rev this to 1.2

Reply:
Got it.  Just didn't know if you wanted my work being a whole version step up.